app/controllers/api/v1/room_settings_controller.rb

```diff
@@ -34,6 +42,21 @@ def find_room
       def room_setting_params
         params.require(:room_setting).permit(:settingValue, :settingName)
       end
+
+      def infer_access_code(value:)
+        # Access code update logic maps 'false' -> removing a code & <OTHER> -> generating a code.
+        case value
+        when 'false'
+          '' # Removing access code.
+        else
+          generate_code # By default, authorized req will (Re)generate access code.
+        end
```

Comment:
no need for the `case` anymore

```
def infer_access_code(value:)
  return '' if value == 'false'
  
  generate_code
end

```

Reply:
You're right,
Do you like further simplifying it into a one line with:
```
def infer_access_code(value:)
  value == 'false' ? '' : generate_code
end
```

Reply:
Yeah that works